test/sequential/bootstrap.updatePromotion.test.ts

```diff
@@ -0,0 +1,214 @@
+/* eslint-disable jest/no-conditional-expect */
+/*
+ *
+ * @group bootstrap
+ * @group sequential
+ */
+import { getApi, initApi } from "../../utils/api";
+import {
+  scheduleBootstrap,
+  finalizeBootstrap,
+  updatePromoteBootstrapPool,
+  provisionBootstrap,
+  claimRewardsBootstrap,
+  getLiquidityAssetId,
+  getBalanceOfAsset,
+} from "../../utils/tx";
+import { EventResult, ExtrinsicResult } from "../../utils/eventListeners";
+import { Keyring } from "@polkadot/api";
+import { User } from "../../utils/User";
+import {
+  getEnvironmentRequiredVars,
+  waitForBootstrapStatus,
+} from "../../utils/utils";
+import {
+  getEventResultFromMangataTx,
+  getEventErrorfromSudo,
+} from "../../utils/txHandler";
+import {
+  checkLastBootstrapFinalized,
+  createNewBootstrapCurrency,
+  setupBootstrapTokensBalance,
+} from "../../utils/Bootstrap";
+import { MGA_ASSET_ID } from "../../utils/Constants";
+import { BN, MangataGenericEvent } from "@mangata-finance/sdk";
+import { setupUsers } from "../../utils/setup";
+
+jest.spyOn(console, "log").mockImplementation(jest.fn());
+jest.spyOn(console, "error").mockImplementation(jest.fn());
+jest.setTimeout(3500000);
+process.env.NODE_ENV = "test";
```

Comment:
why this? :upside_down_face:

Reply:
I deleted this, but I need a comment. Pls check DMs